slash_commands/misc.py

```diff
@@ -134,35 +150,66 @@ async def github_leaderboard(ctx: discord.Interaction, order: str = "commits"):
             f"👥 Followers: {user['followers']}"
             for i, user in enumerate(leaderboard_data[:10])
         ]
-        embed.add_field(name="Top 10", value="\n\n".join(leaderboard_text[:5]), inline=False)
+        embed.add_field(
+            name="Top 10", value="\n\n".join(leaderboard_text[:5]), inline=False
+        )
         embed.add_field(name="", value="\n\n".join(leaderboard_text[5:]), inline=False)
 
     embed.add_field(
         name="🔗 Want to join the leaderboard? Sign up below",
         value=f"[interview.swecc.org](https://interview.swecc.org)",
-        inline=False
+        inline=False,
     )
 
     await ctx.response.send_message(embed=embed, ephemeral=bot_context.ephemeral)
 
+
 async def next_meeting(ctx: discord.Interaction):
     meeting_info = await calendar.get_next_meeting()
     calendar_hyperlink = f"[SWECC Public Calendar]({calendar.get_url()})"
     if meeting_info:
         embed = discord.Embed(
             title=f"📅 Next Meeting: {meeting_info['name']}",
             description=f"**Description:** {meeting_info['description'] or 'No description available.'}\n\n",
-            color=discord.Color.blue()
+            color=discord.Color.blue(),
         )
 
-        embed.add_field(name="🕒 Date & Time", value=meeting_info['date'], inline=False)
-        embed.add_field(name="📍 Location", value=meeting_info['location'] or "Not specified", inline=False)
+        embed.add_field(name="🕒 Date & Time", value=meeting_info["date"], inline=False)
+        embed.add_field(
+            name="📍 Location",
+            value=meeting_info["location"] or "Not specified",
+            inline=False,
+        )
         embed.add_field(name="🔗 Calendar Link", value=calendar_hyperlink, inline=False)
-        
+
         await ctx.response.send_message(embed=embed)
     else:
         await ctx.response.send_message("No upcoming meetings found.")
 
+
+async def attend(ctx: discord.Interaction, session_key: str):
+    status, data = await swecc_api.attend_event(ctx.user.id, session_key)
+
+    if status == 200:
+        embed = discord.Embed(
+            title="Success!",
+            description=f"Your attendance was successfully registered!",
```

Comment:
do you want to add the title of the event here? LGTM otherwise

Reply:
I don't want to add the session key since if other people see it they could attend the meeting without actually being present.

Reply:
the attendance object also has a "title" on top of the key. It looks like the endpoint just returns a 201 right now though

Reply:
maybe we could look into adding it later

Reply:
Yeah for sure